webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptaddMemberToSiteWithRoleBatchTest.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (c) 2005 The Apereo Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://opensource.org/licenses/ecl2
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sakaiproject.webservices;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.withSettings;
+
+
+import org.apache.cxf.jaxrs.client.WebClient;
+import org.junit.Test;
+import org.sakaiproject.tool.api.Session;
+import org.sakaiproject.entity.api.ResourcePropertiesEdit;
+import org.sakaiproject.user.api.User;
+import org.sakaiproject.site.api.Site;
+import org.junit.rules.ExpectedException;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.lang.RuntimeException;
+
+public class SakaiScriptAddMemberToSiteWithRoleBatchTest extends AbstractCXFTest {
```

Comment:
I don't know why... In my local the file has the Add with "A" and not with "a" but I to amend and commit but it says there are not changes.  I'll take a look in google to view if I need to do something with GIT to send the change in the name of the file.